### PR TITLE
[IMP] documentation: improve commands documentation

### PR DESCRIPTION
**Pull request by LucasLefevre**
Task: 0

## Description:

description of this task, what is implemented and why it is implemented that way.

Task: : [TASK_ID](https://www.odoo.com/web#id=TASK_ID&action=333&active_id=2328&model=project.task&view_type=form&cids=1&menu_id=4720)

## review checklist

- [ ] feature is organized in plugin, or UI components
- [ ] support of duplicate sheet (deep copy)
- [ ] in model/core: ranges are Range object, and can be adapted (adaptRanges)
- [ ] in model/UI: ranges are strings (to show the user)
- [ ] undo-able commands (uses this.history.update)
- [ ] multiuser-able commands (has inverse commands and transformations where needed)
- [ ] new/updated/removed commands are documented
- [ ] exportable in excel
- [ ] translations (\_t("qmsdf %s", abc))
- [ ] unit tested
- [ ] clean commented code
- [ ] track breaking changes
- [ ] doc is rebuild (npm run doc)
- [ ] status is correct in Odoo

### CODE REVIEW

File: doc/extending/command.md [hunk 1]
```diff
@@ -1,60 +1,89 @@
...
-1. `LocalCommands`: every other command
-   - manipulate the local state
-   - can be converted into CoreCommands
-   - are not shared in collaborative environment
+### LocalCommand
 
-For example, "RESIZE_COLUMNS_ROWS" is a CoreCommand. "AUTORESIZE_COLUMNS" can be (locally) converted into a "RESIZE_COLUMNS_ROWS", and therefore, is not a CoreCommand.
-CoreCommands should be "device agnostic". This means that they should contain all the information necessary to perform their job. Local commands can use inferred information from the local internal state, such as the active sheet.
+- **Purpose**: Higher-level commands for managing the UI state or dispatching low level core commands.
+- **Handling**: Handle UI plugins. They cannot be handled by core plugins
```
Comment (Rachico) [inline doc/extending/command.md:18]:
```suggestion
- **Handling**: Managed by UI plugins. They cannot be handled by core plugins
```

**Comment by Rachico** [comment]
Much clearer and comprehensive now, thanks!

File: doc/extending/command.md [hunk 2]
```diff
@@ -72,16 +101,17 @@ export function repeatAddColumnsRowsCommand(
...
+import { repeatLocalCommandTransformRegistry, genericRepeat } from "@odoo/o-spreadsheet";
+
 repeatLocalCommandTransformRegistry.add("MY_LOCAL_COMMAND", genericRepeat);
 ```
 
-The difference with the core commands lies in the presence of a third argument in the transformation function: the core commands that were dispatched during the handling of the local command. This is useful if the result of the command depends on the state of an UI Plugin. In this case there's no guarantee that the state of the UI Plugin will be the same when the command is repeated. Adapting the child core commands is then sometimes a valid way to adapt the local command, as they don't depend on any internal state.
+For local commands, the transformation function includes a third argument: the core (sub)commands dispatched during the handling of the root local command. This is useful if the result depends on the UI plugin's state, as there is no guarantee that the UI plugin's state will be the same when the command is repeated. Adapting the child core commands can be a valid way to adjust the local command, as they do not depend on any internal state.
```
Comment (rrahir) [inline doc/extending/command.md:114]:
```suggestion
For local commands, the transformation function includes a third argument: the core (sub)commands dispatched during the handling of the root local command. This is useful if the result depends on the UI plugins' state, as there is no guarantee that the UI plugins' state will be the same when the command is repeated. Adapting the child core commands can be a valid way to adjust the local command, as they do not depend on any internal state.
```
assuming we are encompassing all ui plugins

**Comment by rrahir** [comment]
@robodoo r+

**Pull request closed without merging**